File: exercicioFixacao/listaImutavel.py
def somar(x):
    return x + 1

numeros = [2, 4, 6, 8, 10, 12, 14]

# Manipulação dos elementos da lista sem alterá-la (Imutabilidade)
novosNumeros = list(map(somar, numeros))

print(numeros)
print(novosNumeros)

File: exercicioFixacao/listaImutavel.hs
somar :: Int -> Int
somar x = x + 1

numeros :: [Int]
numeros = [1, 3, 5, 7, 9, 11, 13]

main = do
    -- Manipulação dos elementos da lista sem alterá-la (Imutabilidade)
    let novosNumeros = map somar numeros

    print numeros
    print novosNumeros